pkg/syncthing/api.go

```diff
@@ -50,7 +51,7 @@ func (s *Syncthing) APICall(ctx context.Context, url, method string, code int, p
 		if err == nil {
 			return result, nil
 		}
-		if retries == 3 {
+		if retries == 3 || strings.Contains(url, "rest/db/ignores") {
```

Comment:
why do we need this? could we instead control this on the caller of APICall? (maybe include a param that controls the retries?)